vendor/github.com/bazelbuild/buildtools/build/build_defs.bzl

```diff
@@ -17,36 +17,35 @@ distributed under the License is distributed on an "AS IS" BASIS,
 
 _GO_YACC_TOOL = "@org_golang_x_tools//cmd/goyacc"
 
-def go_yacc(src, out, visibility=None):
-  """Runs go tool yacc -o $out $src."""
-  native.genrule(
-      name = src + ".go_yacc",
-      srcs = [src],
-      outs = [out],
-      tools = [_GO_YACC_TOOL],
-      cmd = ("export GOROOT=$$(dirname $(location " + _GO_YACC_TOOL + "))/..;" +
-             " $(location " + _GO_YACC_TOOL + ") " +
-             " -o $(location " + out + ") $(SRCS)"),
-      visibility = visibility,
-      local = 1,
-  )
+def go_yacc(src, out, visibility = None):
```

Comment:
Please revert this file. This is part of a local copy of `github.com/bazelbuild/buildtools`, so any changes here may be erased when we update to a newer version.

At the moment, Gazelle uses a specific older version of this library since the API broke a couple times. I would guess it's safe to upgrade now though.